github3/github.py

```diff
@@ -1267,11 +1268,17 @@ def project_column(self, number):
                 url, headers=projects.Project.CUSTOM_HEADERS), 200)
         return self._instance_or_null(projects.ProjectColumn, json)
 
-    def public_gists(self, number=-1, etag=None):
+    def public_gists(self, since=None, number=-1, etag=None):
```

Comment:
We can't use keyword-only arguments, unfortunately, so this must show up after `number` and `etag`. People could be passing those positionally and this would break them making this a backwards incompatible change.

Reply:
Makes sense! Updated so that the `since` parameter is at the end.